test/tasks/demo.test.js

```diff
@@ -162,5 +162,27 @@ describe('Demo task', function() {
 
 			demoStream.resume();
 		});
+
+		it('should load local partials', function(done) {
+			fs.mkdirSync('demos/src/partials');
+			fs.writeFileSync('demos/src/test1.mustache', '<div>test1</div>{{>partial1}}', 'utf8');
+			fs.writeFileSync('demos/src/partial1.mustache', '<div>partial1</div>', 'utf8');
+			fs.writeFileSync('demos/src/test2.mustache', '<div>test1</div>{{>partials/partial2}}', 'utf8');
+			fs.writeFileSync('demos/src/partials/partial2.mustache', '<div>partial2</div>', 'utf8');
```

Comment:
Actually because all the tests do this, I'd have to refactor the whole thing for it to work. Otherwise the test files I need get overwritten by previous tests.